### PR TITLE
add ema and EfficientNet

### PR DESCRIPTION
**Pull request by shippingwang**
This PR
- add EMA(ExponentialMovingAverage)
- add EfficientNet (configs, support some params: use_aa， padding_type, etc.)

image

### CODE REVIEW

File: tools/program.py [hunk 1]
```diff
@@ -101,6 +101,8 @@ def create_model(architecture, image, classes_num):
     """
     name = architecture["name"]
     params = architecture.get("params", {})
+    params['is_test'] = not is_train
+    print(params)
```
Comment (WuHaobo) [inline tools/program.py:105]: 统一用logger是不是好些

File: tools/ema.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+# limitations under the License.
+
+
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import paddle
```
Comment (WuHaobo) [inline tools/ema.py:22]: not use

File: ppcls/modeling/architectures/efficientnet.py [hunk 3]
```diff
@@ -383,7 +383,9 @@ def se_block(self, inputs, num_squeezed_channels, oup, name):
             use_bias=True,
             padding_type=self.padding_type,
             name=name + '_se_expand')
-        se_out = inputs * fluid.layers.sigmoid(x_squeezed)
+        #se_out = inputs * fluid.layers.sigmoid(x_squeezed)
```
Comment (littletomatodonkey) [inline ppcls/modeling/architectures/efficientnet.py:386]: remove if not used.

File: tools/program.py [hunk 4]
```diff
@@ -401,6 +418,7 @@ def run(dataloader, exe, program, fetchs, epoch=0, mode='train', vdl_writer=None
 
     Returns:
     """
+    print(fetchs)
```
Comment (littletomatodonkey) [inline tools/program.py:421]: remove it?

Comment (WuHaobo) [inline tools/program.py:421]: logger

Comment (shippingwang) [reply tools/program.py:421]: > remove it?

yep

File: ppcls/modeling/architectures/efficientnet.py (hunk 3 above)
Comment (shippingwang) [reply ppcls/modeling/architectures/efficientnet.py:386]: > remove if not used.

yep

**Pull request merged**